platform/app/.recipes/Nginx-Orthanc-Keycloak/.env.example

```
@@ -0,0 +1,8 @@
+# Copy this file to .env next to docker-compose.yml and fill in strong,
+# unique values. docker compose refuses to start while any of these is unset.
+#
+# POSTGRES_PASSWORD is the single PostgreSQL credential: it provisions the
+# keycloak database role and Keycloak reuses it to connect (KC_DB_PASSWORD is
+# derived from it in docker-compose.yml), so there is one value, not two.
+POSTGRES_PASSWORD=
+KEYCLOAK_ADMIN_PASSWORD=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The KEYCLOAK_ADMIN_PASSWORD key should go before the POSTGRES_PASSWORD key

(UnorderedKey)